.github/workflows/acceptance-tests.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
       name: renku-ci-gr-${{ github.event.number }}
     steps:
       - name: deploy-pr
-        uses: SwissDataScienceCenter/renku-actions/deploy-renku@v1.6.0
+        uses: SwissDataScienceCenter/renku-actions/deploy-renku@feat-combined-helm-charts
```

Comment:
I believe the `feat-combined-helm-charts` is something temporal and will be updated with a new release of renku-actions. Is that the intention?

Reply:
Now it is 1.7.0